config/webhook/manifests.yaml

```diff
@@ -0,0 +1,29 @@
+
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  creationTimestamp: null
+  name: validating-webhook-configuration
+webhooks:
+- admissionReviewVersions:
+  - v1
+  - v1beta1
+  clientConfig:
```

Comment:
We need to think of how to guide non-OLM users to add `caBundle` here, a kustomize patch?
BTW I hope that CA bundle is managed automagically by OLM.

Reply:
>  BTW I hope that CA bundle is managed automagically by OLM.
It [does](https://github.com/openshift/external-dns-operator/pull/31#discussion_r722233346)